server/src/assembly/resources/conf/iotdb-datanode.properties

```diff
@@ -178,6 +178,11 @@ target_config_nodes=127.0.0.1:22277
 # Datatype: long
 # delete_wal_files_period_in_ms=20000
 
+# Maximum size of wal buffer used in MultiLeader Consensus
+# If it's a value smaller than 0, use the default value 10 * 1024 * 1024 * 1024 bytes (10GB).
+# Datatype: long
+# multiLeader_max_wal_buffer_size_in_byte=10737418240
```

Comment:
The description is inconsistent with the code. In fact, when the wal size exceeds this value, the writing will slow down. And the final stored wal size may exceed this value. I think we can revise the description to avoid misunderstanding.

Reply:
And i think we can change the `multiLeader_max_wal_buffer_size_in_byte` to `multiLeader_max_wal_size_in_byte` becase the `multiLeader_max_wal_buffer_size_in_byte` may confuse people with the `wal_buffer_size_in_byte`

Reply:
Another suggestion is that we can use a more concrete name for this parameter according to our scenario such as `multi_leader_throttle_down_threshold_by_wal`

Reply:
Get it, I will fixed it by using `multi_leader_throttle_down_threshold_in_byte`